### ISSUE TITLE
Lists - polynomial exercise - I'm not sure this is accessible to many of our learners

### ISSUE THREAD

**Issue by KateCourt** [carpentries-incubator/python-novice-programming-gapminder#23]
<details>
<summary><strong>Instructions</strong></summary>

Thanks for contributing! :heart:

If this contribution is for instructor training, please email the link to this contribution to
checkout@carpentries.org so we can record your progress. You've completed your contribution
step for instructor checkout by submitting this contribution!

If this issue is about a specific episode within a lesson, please provide its link or filename.

Keep in mind that **lesson maintainers are volunteers** and it may take them some time to
respond to your contribution. Although not all contributions can be incorporated into the lesson
materials, we appreciate your time and effort to improve the curriculum. If you have any questions
about the lesson maintenance process or would like to volunteer your time as a contribution
reviewer, please contact The Carpentries Team at team@carpentries.org.

You may delete these instructions from your comment.

\- The Carpentries
</details>

**Comment by Fehings** [comment]
I think this could be made a little more accessible with changing the comments to just say we have a mathematical function in the form y = a + bx + cx^2. Would that make this less daunting?
It is a nice use of enumerate, but I do see your point!

**Comment by KateCourt** [comment]
Yeah, that'd help. For non maths background people though I think you'd still need to be prepared to spend a little longer explaining the equation and some definitions. But that's not a problem in itself, maybe could add some notes for non maths-y instructors on how best to explain it!
